backend/src/managers/TranslationManager.ts

```diff
@@ -96,14 +98,22 @@ export default class TranslationManager {
         return [PRE_PROMPT + role + POST_PROMPT, hint];
     }
 
-    static interpreterString(prompt: string, content: string): APIPromise<Stream<ChatCompletionChunk>> {
+    static interpreterString(prompt: string, content: string, temperature: number): APIPromise<Stream<ChatCompletionChunk>> {
         return openai.chat.completions.create({
             messages: [
-                {role: 'system', content: prompt},
-                {role: 'user', content: content}
+                // let's duplicate the instructions for better emphasis
+                {role: 'system', content: (prompt + ' Текст ниже:')},
+                {role: 'user', content: content},
+                {role: 'system', content: ('Текст выше. Напоминаю инструкцию:\n' + prompt)},
+                {
+                    role: 'system', content: 'Answer MUST CONTAIN ONLY THE MODIFIED TEXT ' +
+                        'as if written by the original author. PRESERVE the original HTML tags and formatting AS IS.\n' +
```

Comment:
This is excessive - you are stripping all HTML from the text.

Reply:
I'm not stripping html (only for the noop check).